tools/deprecate_version/deprecate_version.py

```diff
@@ -90,47 +90,61 @@ def CreateIssues(access_token, runtime_and_pr):
     raise DeprecateVersionError('Unknown labels (expected %s, got %s)' % (LABELS, labels))
 
   issues = []
-  for runtime_guard, pr in runtime_and_pr:
+  for runtime_guard, pr, commit in runtime_and_pr:
     # Who is the author?
-    pr_info = repo.get_pull(pr)
+    if pr:
+      # Extract PR title, number, and author.
+      pr_info = repo.get_pull(pr)
+      change_title = pr_info.title
+      number = ('#%d') % pr
+      login = pr_info.user.login
+    else:
+      # Extract Commit message, sha, and author.
+      # In this case we cannot add a user login for an assignee.
+      change_title = commit.message.split('\n')[0] # Remove sign-off mesage
+      number = ('commit %s') % commit.hexsha
+      login = None
```

Comment:
GitPython API can only give me author name and email, so I could not assign the issue to the NamedUser login like before. GitPython experts, is there a way?

Reply:
Not that I know of.  Does assigning to email work?  Worst case we could assign the script runner to find an owner and print that's what is going on, but it'd be nice if we could do better.

Reply:
I found `search_users` functionality in the API -- I did guard against not finding the user, but I think (?) there's guarantee there will be a user with the email we find.

Reply:
Edit: I trial ran it without creating the issue, I successfully got your login with the script now:
```
envoy.reloadable_features.http1_flood_protection deprecation
commit f5b0294f0a7c2d143f6d4d94afae9a0e6f9dacf7 (http: adding response flood protection) introduced a runtime guarded feature. This issue tracks source code cleanup.
  >> Assigning to alyssawilk
```